models/components/components.go

```diff
@@ -0,0 +1,54 @@
+/*
+ Copyright 2024 The OSCAL Compass Authors
+ SPDX-License-Identifier: Apache-2.0
+*/
+
+package components
+
+import (
+	oscaltypes112 "github.com/defenseunicorns/go-oscal/src/types/oscal-1-1-2"
```

Comment:
Just wondering if there is benefit to leaving out the `112` in the import alias name to simplify updating to newer versions in the future?

Reply:
Good point @gvauter. I will fix on subsequent PRs to the branch.